modules/iam-assumable-role-with-saml/outputs.tf

```diff
@@ -1,19 +1,19 @@
 output "iam_role_arn" {
   description = "ARN of IAM role"
-  value       = element(concat(aws_iam_role.this.*.arn, [""]), 0)
+  value       = try(aws_iam_role.this[0].arn, "")
```

Comment:
Is this the official way of dealing with attribute outputs from resources that may or may not exist now? I like it, it's cleaner, but it works in all cases, like when on `destroy` also?

Reply:
official - I don't know. however, I definitely use and abuse `try()` since it works so well in various use cases; outputs just happens to be one of those (and it shortens up the syntax a bit)

Reply:
@max-rocket-internet `try()` works better than using ternary operators for `destroy`. `try()` works in the same way as `element(concat(...))` but shorter.